documentation/modules/proc-configuring-kafka-connect-user-authorization.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-connect-configuration.adoc
+
+[id='proc-configuring-kafka-connect-user-authorization-{context}']
+= Configuring Kafka Connect user authorization
+
+This procedure describes how to authorize user access to Kafka Connect.
+
+When simple authorization is being used to access Kafka connect, a Kafka Connect user requires access rights to the consumer group
+and the internal topics of Kafka Connect.
+
+Simple authorization uses ACL rules to provide the right level of access, which are handled by the Kafka `SimpleAclAuthorizer` plugin.
```

Comment:
Again, I think we should clarify that this example is for use with `simple` authorization, but that other authorization types will also need to give the rights to the user.

Reply:
> When **any type** of authorization is being used in Kafka, a Kafka Connect user requires access rights to the consumer group and the internal topics of Kafka Connect.
> 
> In this procedure we show how access is provided when `simple` authorization is being used.